### DISCUSSION TITLE
Add a feature voting site for the community

### DISCUSSION THREAD

**Discussion by ptrsmk** [community-scripts/ProxmoxVE#6] [Ideas]
It seems to me that a proper ~~feedback~~ feature voting site would help manage and prioritize community requests.  Utilizing the Discussions on Github gets unwieldly, and it is more likely to result in lots of duplicate requests with no clear idea of how popular an idea is.  Additionally, a centralized, dedicated feedback site has the benefit of acting as a repository of self-hosted projects for discovery.

To be clear, this would not be a replacement for utilizing Discussions or the Issue tracker, and that would be made explicit on the site.

I'm thinking that we could utilize [Fider](https://fider.io/) or [ClearFlask](https://clearflask.com/) or [Astuto](https://astuto.io/), all of which are open source.

**Comment by judeibe** [comment]
I do think is valuable, one to keep all the centrally located and two to not do any that has cost. Like hosting a server even if it's self hosted.

I think what is needed are roles like moderators, etc...

**Comment by MickLesk** [comment]
I've thought of something similar before, I wanted to see if GitHub offers something like this directly. (Marketplace)

The biggest drawback is that it's hosted externally, which means twice the work and twice the moderation. 

Or we create such a thing on the Website @BramSuurdje ?

**Reply by judeibe** [reply]
Any site using GitHub pages will be static, so not interactivity/state. Discussions does allow for voting using Ideas. 1 per script. A moderator would have to keep duplicates in check.

**Reply by MickLesk** [reply]
Hard. Okay, then i improve the discussion section next week.

**Reply by bramsuurdje** [reply]
a dedicated discussion with a proper template will do in my opinion. like @judeibe said, the one downside of GH-Pages is that all the sites are static. something like the following would do nicely as a template

```
# Script Request

- App Name:
- Purpose:
- Website:
```

**Reply by ptrsmk** [reply]
Considering that a new discussion template was [merged](https://github.com/community-scripts/ProxmoxVE/pull/20) for this purpose, it looks like that is the route that's been chosen.  That said, the new template isn't usable until certain other tasks are performed by the admins -- namely the creation of a new discussion category.
